assets/js/playlist.js

```diff
@@ -59,6 +62,37 @@
       url: $(this).attr('url'),
       data: $(this).serialize()
     }).done(function(response){
-      $('#playlists').html(response)
+      $('#playlists').html(response);
     });
-  });
\ No newline at end of file
+  });
+
+  // ajax for deleting songs
+  $(document).on('submit', '.deleteSong', function( event ){
+    event.preventDefault();
+    $.ajax({
+      method: 'POST',
+      url: $(this).attr('action'),
+      data: $(this).serialize()
+    }).done(function(response){
+      //remove song from the template
+      $("#"+response.song_id).parent( "li" ).remove()
+      song_ids.pop();
+      console.log(song_ids);
+    });
+  });
+
+  // function for creating csrf_token 
+  function getCookie(name) {
```

Comment:
where do you get this? you already pass csrf_token from your form, check your form serializer.

Reply:
remove this.

Reply:
I will change it, sir. I found that here 
https://docs.djangoproject.com/en/1.11/ref/csrf/